common/djangoapps/enrollment/views.py

```diff
@@ -71,7 +71,12 @@ class EnrollmentUserThrottle(UserRateThrottle, ApiKeyPermissionMixIn):
     rate = '40/minute'
 
     def allow_request(self, request, view):
-        return self.has_api_key_permissions(request) or super(EnrollmentUserThrottle, self).allow_request(request, view)
+        user = request.user
+
+        if user.is_superuser or user.is_staff:
```

Comment:
This also goes against the request of DevOps. Why not update this throttle to allow a higher rate for staff?

```python
from rest_framework.throttling import UserRateThrottle


class EnrollmentUserThrottle(ApiKeyPermissionMixIn, UserRateThrottle):
    THROTTLE_RATES = {
        'user': '40/minute',
        'staff': '200/minute',
    }

    def allow_request(self, request, view):
        # Use a special scope for staff to allow for a separate throttle rate
        user = request.user
        if user.is_authenticated() and (user.is_staff or user.is_superuser):
            self.scope = 'staff'
            self.rate = self.get_rate()
            self.num_requests, self.duration = self.parse_rate(self.rate)

        return super(EnrollmentUserThrottle, self).allow_request(request, view)

```